assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -182,69 +222,97 @@ function MediaPane(props) {
     )
     .map((attachment) => getResourceFromAttachment(attachment));
 
+  const refContainer = useRef();
+  const refContainerFooter = useRef();
+
+  useIntersectionEffect(
+    refContainerFooter,
+    (entry) => {
```

Comment:
For effect hooks I prefer that the callback and deps go as the last two arguments.

Reply:
I was thinking about `options` as a last optional argument, so you can skip them most of the time, but maybe it's a wrong assumption (we are using them here).

Reply:
We'll just have to pass `{}` for now. I'm hoping that the React "exhaustive deps" linter will eventually figure out how to do this kind of hooks, and they will really only have two options: first two args are callback and deps, or last two args are. And I think last two args are more likely for them.